webview-ui/src/components/GetSolutionDropdown.tsx

```diff
@@ -84,14 +84,17 @@ const GetSolutionDropdown: React.FC<GetSolutionDropdownProps> = ({ incidents, sc
           <DropdownItem key="get-rag-solution" onClick={() => onGetSolution(incidents)}>
             Get solution
           </DropdownItem>
-          {scope === "incident" && incidents.length === 1 && state.isContinueInstalled && (
-            <DropdownItem
-              key="ask-continue-konveyor"
-              onClick={() => onGetSolutionWithKonveyorContext(incidents[0])}
-            >
-              Ask Continue with Konveyor Context
-            </DropdownItem>
-          )}
+          {scope === "incident" &&
+            incidents.length === 1 &&
+            state.isContinueInstalled &&
+            isKonveyor() && (
```

Comment:
I think we should still have this in the "downstream" thing. I wouldn't gate this.